pom.xml

```diff
@@ -656,6 +656,18 @@
                     <artifactId>license-maven-plugin</artifactId>
                     <version>2.0.0</version>
                     <configuration>
+                        <!-- Some Maven artifacts do not explicitly state the
+                        license under which they are published. At this time
+                        the following dependencies are explicitly excluded from
+                        the license validation process:
+                        - org.osgi.enterprise: The newer version of this
+                          software is licensed as Apache-2.0, see
+                          https://repo1.maven.org/maven2/org/osgi/org.osgi.enterprise/5.0.0/org.osgi.enterprise-5.0.0.pom.
```

Comment:
Ah, what we internally do in this case is to explicitly bump the managed dependency version. Would that work here as well?

Reply:
I couldn't see it in `dependencyManagement`. Can you point me the place we do that?

Reply:
I added a commit to the relevant repo (including other fixes necessary for this upgrade); PTAL :)